e2e_samples/parking_sensors_synapse/synapse/workspace/linkedService/sywsdevimgbb-WorkspaceDefaultSqlServer.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "sywsdevimgbb-WorkspaceDefaultSqlServer",
+	"type": "Microsoft.Synapse/workspaces/linkedservices",
+	"properties": {
+		"typeProperties": {
+			"connectionString": "Data Source=tcp:sywsdevimgbb.sql.azuresynapse.net,1433;Initial Catalog=@{linkedService().DBName}"
```

Comment:
You'll want to delete this file as it has workspace-specific (sywsdevimgbbb) which doesn't get replaced during the deployment script.